posts/greencouchpotato/2022-10-06-first-clicky-turtle.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: posts
+author:greencouchpotato
+title: "Clicky turtle hack"
+---
```

Comment:
Hi Rosalind, I see the format you've requested already added. Could you please re-review and let me know if I'm missing something?